ssl-config/src/main/scala/com/avast/sst/ssl/SslContextModule.scala

```diff
@@ -14,11 +14,14 @@ import scala.language.higherKinds
 
 object SslContextModule {
 
+  private[this] val referenceConfig = ConfigFactory.defaultReference().getConfig("ssl-config")
```

Comment:
This should be inside `Sync[F].delay` or `lazy`.

Reply:
That means we need to move it inside `make`. I'm OK with that, I just think it's not worth it.

Reply:
Yes, this will be called only once most probably so let's be pure in this.

Reply:
moved into make